agr_literature_service/lit_processing/dump_json_data.py

```diff
@@ -526,24 +498,24 @@ def get_journal_data(db_session):
 
 def print_usage():
```

Comment:
Argparse has its own print_help() function and we can avoid defining an additional one. If the only reason to add this function is to force `mod` to be in a predefined list of allowed values, we can use the `choices` option for add_argument(): https://docs.python.org/3/library/argparse.html#the-add-argument-method